src/field_theory/galois.lean

```diff
@@ -43,7 +43,19 @@ section
 variables (F : Type*) [field F] (E : Type*) [field E] [algebra F E]
 
 /-- A field extension E/F is galois if it is both separable and normal -/
-@[class] def is_galois : Prop := is_separable F E ∧ normal F E
+class is_galois : Prop :=
+(separable' : is_separable F E)
+(normal' : normal F E)
+
+variables {F E}
+
+theorem is_galois_iff : is_galois F E ↔ is_separable F E ∧ normal F E :=
+⟨λ h, ⟨h.1, h.2⟩, λ h, ⟨h.1, h.2⟩⟩
+
+theorem is_galois.sep (h : is_galois F E) : is_separable F E := h.1
+theorem is_galois.norm (h : is_galois F E) : normal F E := h.2
```

Comment:
Can you use `normal` instead? `norm` has a completely different meaning, and is also relevant in field extensions.

Reply:
I think `normal` is also taken, for the unfolded form of the same theorem. The use of shortened names for fields of the structure is somewhat common, so I was going for that.